graderutils/tracebackformat.py

```diff
@@ -37,67 +37,70 @@ def _iter_redacted_lines(lines, remove_lines, replacement_string):
         yield line
 
 
-def hide_exception_traceback(output, exception_class_name, remove_more_sentinel=None, replacement_string=None):
+def hide_exception_traceback(output, exception_class_name, hide_tracebacks=True, remove_sentinel=None, replacement_string=None):
```

Comment:
The default for `hide_tracebacks` is different here than when reading it from the config. Is that important?

Reply:
> The default for `hide_tracebacks` is different here than when reading it from the config. Is that important?

The `baseconfig.yaml` sets `hide_tracebacks` to `true` by default too.
https://github.com/Aalto-LeTech/python-grader-utils/blob/7baddd5e6255ccbc0868d9a871b29a6e7155cfbc/graderutils/baseconfig.yaml#L7

Reply:
If there is no need to have it as optional for the function API (i.e. all places that use it always pass the value).